### PR TITLE
Update links, add clarification for isPostalInCity()

### PR DESCRIPTION
**Pull request by PatrickCronin**

### CODE REVIEW

File: lib/WebService/MinFraud/Record/BillingAddress.pm [hunk 1]
```diff
@@ -49,7 +49,7 @@ country as that of the IP address.
 =head2 is_postal_in_city
 
 Returns a boolean indicating whether the billing postal code is in the
-billing city.
+billing city, or an undef if the address is not in USA.
```
Comment (oschwald) [inline lib/WebService/MinFraud/Record/BillingAddress.pm]: I think this one is probably phrased as something like "The attribute will not be set if the address is not in the USA." We provide predicate methods so the user can check whether it has been set.

File: lib/WebService/MinFraud/Record/ShippingAddress.pm [hunk 2]
```diff
@@ -75,7 +75,7 @@ country as that of the IP address.
 =head2 is_postal_in_city
 
 Returns a boolean indicating whether the shipping postal code is in the
-shipping city.
+shipping city, or an undef if the address is not in USA.
```
Comment (oschwald) [inline lib/WebService/MinFraud/Record/ShippingAddress.pm]: Same.

**Review by oschwald — CHANGES_REQUESTED** [review]

File: lib/WebService/MinFraud/Record/BillingAddress.pm (hunk 1 above)
Comment (PatrickCronin) [reply lib/WebService/MinFraud/Record/BillingAddress.pm]: Sounds good! Fixed.

File: lib/WebService/MinFraud/Record/ShippingAddress.pm (hunk 2 above)
Comment (PatrickCronin) [reply lib/WebService/MinFraud/Record/ShippingAddress.pm]: Sounds good! Fixed.

**Pull request merged**